template-for-repository/.gitignore

```diff
@@ -3,6 +3,7 @@ proofs/**/logs
 proofs/**/gotos
 proofs/**/report
 proofs/**/html
+proofs/output
```

Comment:
What is the difference between the output directory and the report directories?  Will the report for an individual proof for 'make report' go into the report directory?  Will the report for run-cbmc-proofs.py go into the output directory?  This feature seemed to be geared toward CI (or the person simulating CI) and not to the day-to-day developer of the proof.  Is it the case that all this pull request is doing is creating a symbolic link from the proofs directory to the /tmp directory containing the litani report?  I don't have a good experience with using the litani report while the script is running, because the links change so rapidly that the files are gone by the time I try to load them.

Reply:
See "Document the presence of the `output` directory", hopefully that clarifies some of this.

> What is the difference between the output directory and the report directories?

Report directories are where CBMC-Viewer writes its output. They are opaque to Litani.
`output` is where Litani writes all of its files. Each run of Litani will be a subdirectory of `output`, and in addition, the directory `output/latest` will be a symbolic link to the most recent such subdirectory. So:
```
\_output
| |
| \_ 2021-05-21T10:00:00
| | |_ html
| | |_ run.json
| | \_ [other irrelevant litani stuff]
| |
| \_ 2021-05-21T10:30:00
| | |_ html
| | \_ run.json
| |
| \_ 2021-05-21T10:40:00
| | |_ html
| | \_ run.json
| |
| \_ latest --> 2021-05-21T10:40:00
```

Reply:
I wrote a new comment in the epilog below:
```
The litani reports will be located under the `output` directory. The
HTML report from the latest Litani run will always be symlinked to
`output/latest/html/index.html`, so you can keep this page open in
your browser and reload the page whenever you re-run this script.
```

Reply:
The command `git grep 'Document the presence'  .` yields nothing, I don't know what you want me to look at.  Would you mind changing this sentence to "The litani reports will be located under the `output` directory (the cbmc-viewer reports will remain in the `reports` directory)."  The problem is that the word "report" is used all over, and because "make report" has traditionally meant make the cbmc-viewer report, the word report is becoming ambiguous.

Reply:
Ignore this "git grep" comment, you were naming the commit to study.  Thanks.